apps/web/assets/style.scss

```diff
@@ -1,51 +1,57 @@
-:root {
-  --colors-2-primary-50: 236 255 255;
-  --colors-2-primary-100: 206 251 255;
-  --colors-2-primary-200: 164 247 253;
-  --colors-2-primary-300: 101 237 251;
-  --colors-2-primary-400: 32 219 240;
-  --colors-2-primary-500: 4 190 214;
-  --colors-2-primary-600: 6 151 180;
-  --colors-2-primary-700: 12 121 146;
-  --colors-2-primary-800: 20 98 118;
-  --colors-2-primary-900: 21 80 100;
-  --colors-2-primary-950: 7 51 66;
+@import '_variables';
 
-  --colors-2-secondary-50: 255 253 231;
-  --colors-2-secondary-100: 255 252 193;
-  --colors-2-secondary-200: 255 247 135;
-  --colors-2-secondary-300: 255 234 66;
-  --colors-2-secondary-400: 255 217 15;
-  --colors-2-secondary-500: 241 192 2;
-  --colors-2-secondary-600: 207 148 0;
-  --colors-2-secondary-700: 165 105 3;
-  --colors-2-secondary-800: 136 82 11;
-  --colors-2-secondary-900: 116 67 15;
-  --colors-2-secondary-950: 68 34 4;
-}
+@tailwind base;
 
-.scrollbar-hidden {
-  -ms-overflow-style: none;
-  scrollbar-width: none;
+@layer base {
```

Comment:
**issue**: Remove all `@tailwind` and `@layer` directives. Declaring them here causes Tailwind to be imported twice, because Tailwind is also registered as a Nuxt module.